### ISSUE TITLE
Is it possible to expose descendants of childrenArePresentational?

### ISSUE THREAD

**Issue by Malvoz** [w3c/html-aam#292]
The Maps For HTML Community Group has a [custom `<map>` component](https://github.com/Maps4HTML/Web-Map-Custom-Element), but in testing I had found that no child elements of `<map>` (I believe anything that isn't `<area>`) are being exposed in the accessibility tree.

Is it possible to tell UAs to expose descendants of [`childrenArePresentational`](https://www.w3.org/TR/wai-aria-1.1/#childrenArePresentational)?

**Comment by zcorpan** [comment]
Is it specified somewhere that `map` should have `childrenArePresentational`?

**Comment by scottaohara** [comment]
This behavior is not specified in this spec.  Rather, I filed a bug against chromium https://bugs.chromium.org/p/chromium/issues/detail?id=1208405 since Firefox and Safari are working as expected.

Leaving this open for tracking purposes, but the ball should be in Chromium's court to fix this.

**Comment by Malvoz** [comment]
Oops, I forgot to link the chromium bug I filed: https://bugs.chromium.org/p/chromium/issues/detail?id=1147035. Sorry!

In that bug I incorrectly refer to wanting to change the implicit ARIA _role_, I probably meant to say ARIA _semantics_, I corrected that statement in a later comment. I hope it's not too confusing, it seems on their response I got my point across.

However, because it seemingly isn't defined anywhere that `map` should have `childrenArePresentational` I guess it's also incorrect for me to imply in that bug that a `role` is required to opt-out of that behavior. I think I should comment on my bug and refer to yours as superseding mine. Does that sound right to you @scottaohara?

**Comment by Malvoz** [comment]
Because it seemingly isn't defined anywhere that `map` should have `childrenArePresentational` I guess it's also incorrect for me to imply in that bug that a `role` is required to opt-out of that behavior. I think I should comment on my bug and refer to yours as superseding mine.

**Comment by scottaohara** [comment]
yes @Malvoz, please refer to mine.  it's not really even that the children are treated as presentational (eg role=presentation) - if it was then the child content would still be exposed, but have no roles exposed.  The content is rather being flat out ignored. 

From what i saw in testing today, the accessibility panels in chrome were exposing the correct information - but something is still weird since, again, regardless of being exposed in the a11y panel, none of the content is actually accessible to the screen readers when using only chromium browsers

**Comment by Malvoz** [comment]
> I filed a bug against chromium https://bugs.chromium.org/p/chromium/issues/detail?id=1208405

@scottaohara The bug you filed was marked as fixed, though I'm not seeing (hearing) any improvements using NVDA. Perhaps I've misunderstood the bug and the fix, perhaps it should now be considered an NVDA issue that the non-area elements aren't announced in your attached HTML file?

Edit: I took the liberty to re-submit that issue, in which they again have been able to reproduce the behavior: https://issues.chromium.org/issues/419563164

**Comment by scottaohara** [comment]
cc @aleventhal per above comment.  

i haven't looked into it / didn't notice it was marked as fixed.
